src/main/scala/org/ergoplatform/nodeView/state/ErgoState.scala

```diff
@@ -186,10 +176,8 @@ object ErgoState extends ScorexLogging {
   /**
     * Starting bytes for rewardOutputScript
     */
-  def rewardOutputScriptStartBytes(delta: Int): Array[Byte] = delta match {
-    //    case -1000 => Algos.decode("9683020192a39ae4c6a7040505cf0fcd07").get
```

Comment:
This should be replaced to correct values, instead of deleting. But should be done in a separate PR